lcls_tools/common/measurements/screen_beam_profile_measurement.py

```diff
@@ -0,0 +1,99 @@
+from lcls_tools.common.devices.screen import Screen
+from lcls_tools.common.image.processing import ImageProcessor
+from lcls_tools.common.data.fit.projection import ProjectionFit
+from lcls_tools.common.measurements.measurement import Measurement
+import numpy as np
+
+
+class ScreenBeamProfileMeasurement(Measurement):
```

Comment:
Still not sure beam is necessary in this name...or screen either?  A Profile could be measured with a wire as well, but was the thought a wire profile measurement would be separate code?

Reply:
I think this file was written with a screen profile in mind, but it could be generalized.